sdks/python/apache_beam/pvalue.py

```diff
@@ -201,6 +201,43 @@ class PDone(PValue):
   pass
 
 
+class PTuple(object):
+  """An object grouping multiple PCollections.
+
+  This class is useful for returning a named tuple of PCollections from a
+  composite.
+  """
+
+  def __init__(self, pcoll_dict):
+    """Initializes this named tuple with a dictionary of tagged PCollections.
+    """
+    self._pcolls = pcoll_dict
+
+  def __str__(self):
+    return '<%s>' % self._str_internal()
```

Comment:
Remove and let it default to `__repr__`.

Reply:
done